src/devices/Ad7193/Metadata/Ad7193Metadata.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Device.Spi;
+using System.Text;
+
+namespace Ad7193.Metadata
+{
+    /// <summary>
+    /// Metadata class for AD7193
+    /// </summary>
+    public class Ad7193Metadata : IDeviceMetadata, ISpiDeviceMetadata, IAdcDeviceMetadata
+    {
+        /// <inheritdoc/>
+        public string Manufacturer => "Analog Devices";
+
+        /// <inheritdoc/>
+        public string Product => "AD7193";
+
+        /// <inheritdoc/>
+        public string ProductCategory => "ADC";
+
+        /// <inheritdoc/>
+        public string ProductDescription => "4-Channel, 4.8 kHz, Ultralow Noise, 24-Bit Sigma-Delta ADC with PGA";
+
+        /// <inheritdoc/>
+        public string DataSheetURI => "https://www.analog.com/media/en/technical-documentation/data-sheets/AD7193.pdf";
+
+        /// <inheritdoc/>
+        public SpiMode ValidSpiModes => SpiMode.Mode3;
```

Comment:
You should put all SPI settings directly on the ADC class, i.e.:
```
const int DefaultSpiFrequency =  ...;
const SpiMode DefaultSpiMode = ...;
```

I'm not sure about adding validation for Mode3. We can leave this logic for now but some devices are sometimes embedded in others and some settings occasionally change while device is still the same